src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets

```diff
@@ -237,15 +222,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuild
       Projects="@(ValidProjectInputForPackGraph)"
       Targets="_AddMetadataToProjectReference"
-      Properties="%(_MSBuildProjectReferenceExistent.SetConfiguration);
-              %(_MSBuildProjectReferenceExistent.SetPlatform);
-              CustomAfterMicrosoftCommonTargets=$(MSBuildThisFileFullPath);
+      Properties="CustomAfterMicrosoftCommonTargets=$(MSBuildThisFileFullPath);
               BuildProjectReferences=false;
-              TargetFramework=$(TargetFramework);
+              _ParentTargetFramework=$(TargetFramework);
```

Comment:
Ah I misunderstood what this is doing. The only exposure would be if anything _AddMetadataToProjectReferences  were conditional on TargetFramework, and from the list below that seems like a non-scenario. :)

```
        <PackageId>$(PackageId)</PackageId>
        <PackageVersion>$(PackageVersion)</PackageVersion>
        <IncludeAssets>$(IncludeAssets)</IncludeAssets>
        <ExcludeAssets>$(ExcludeAssets)</ExcludeAssets>
        <PrivateAssets>$(PrivateAssets)</PrivateAssets>
```

FWIW, I was confused by the target name. From the POV of the project that actually executes the target, it has nothing to do with enumerating project references.

The point about CustomAfter for cross-targeting stands though. I think you would not get the data from x-targeted libs if that is how they get these targets. Actually, wouldn't they have them already though if they reference the SDK and wouldn't this cause duplicate imports?